nuxt/components/edit/EditDraftLyrics.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    persistent
+    fullscreen
+    no-click-animation
+    hide-overlay
+  >
+    <template #activator="{ on }">
+      <v-btn
+        icon
+        v-on="on"
+      >
+        <v-icon>lyrics</v-icon>
+      </v-btn>
```

Comment:
For some reason, this button and the heart button are showing up as black when I am viewing the app in light mode. Can you check?
<img width="438" alt="image" src="https://github.com/nawhas/nawhas/assets/379169/510b80db-3f10-4826-ab3e-d2114dc7a3b6">

Reply:
Done